recipes/boost/all/conandata.yml

```diff
@@ -34,6 +34,12 @@ sources:
       "https://sourceforge.net/projects/boost/files/boost/1.74.0/boost_1_74_0.tar.bz2"
     ]
     sha256: "83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1"
+  1.75.0:
+    url: [
+      "https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.bz2",
+      "https://sourceforge.net/projects/boost/files/boost/1.75.0/boost_1_75_0.tar.bz2"
+    ]
+    sha256: "953db31e016db7bb207f11432bef7df100516eeb746843fa0486a222e3fd49cb"
 patches:
```

Comment:
Does everything work without patching for 1.75.0? 🎉

Reply:
Everything working? Sadly we're not living in such an utopia haha

I didn't check/port the patches from 1.74 (yet).

Reply:
The patches for 1.74 applied cleanly on 1.75

Reply:
Great.